longabout.html

```diff
@@ -8,41 +8,41 @@
 <body lang="EN-US">
 <h2>About This Content</h2>
 
-<p><em>January 24, 2018</em></p>
+<p><em>October 2, 2019</em></p>
```

Comment:
The terms haven't changed, so the date shouldn't either.

Reply:
Reverted.